check-diff.sh

```diff
@@ -69,10 +70,16 @@ function set_environment_variables {
 			DIFF_BASE=${DIFF_BASE:-$TRAVIS_COMMIT^}
 		fi
 		DIFF_HEAD=${DIFF_HEAD:-$TRAVIS_COMMIT}
+	fi
+
+	if [[ -n "${BITBUCKET_BRANCH+set}" ]]; then
```

Comment:
@westonruter The fi closes out the if statement on [78](https://github.com/glewiswp/wp-dev-lib/blob/136390c5305f827f63d2034d6801da9aec6b7be6/check-diff.sh#L78). This is essentially the fall back (when the environment variables `TRAVIS` and `BITBUCKET_BRANCH ` are not set) which was on the original repo at [72-75](https://github.com/xwp/wp-dev-lib/blob/master/check-diff.sh#L72-L75).

As for `+set` I was under the impression that this was needed to check if a variable was set in bash (I'm no bash master), instead of checking against a value. If it's incorrect I will gladly update.